src/test/transaction_tests.cpp

```diff
@@ -745,7 +745,7 @@ BOOST_AUTO_TEST_CASE(test_IsStandard) {
     t.vout[0].scriptPubKey = CScript() << OP_RETURN;
     BOOST_CHECK(IsStandardTx(CTransaction(t), reason));
 
-    // Only one TxoutType::NULL_DATA permitted in all cases
+    // 2 OP_RETURNs allowed
```

Comment:
This might be worth making a loop. But seems fine.

Reply:
Without loop, a failure will tell you which test failed